src/components/transaction-list/TransactionList.js

```diff
@@ -215,7 +215,6 @@ export default function TransactionList({
         accountName={accountSymbol}
         addCashAvailable={addCashAvailable}
         as={NativeTransactionListView}
-        avatarOptions={avatarOptions}
```

Comment:
dis is spooky, is this not being used anymore?

Reply:
It isn't, checked and it was used for this UIMenu that you spotted above, it was completely unused and I provided screenshots what it used to be a couple of comments above.